### DISCUSSION TITLE
How to customize picker buttons

### DISCUSSION THREAD

**Discussion by dancb10** [fluttercandies/flutter_wechat_assets_picker#98] [Q&A] [answered]
Hello,
I like what you have built there, really impressive.
I've read the docs and looked a bit in the source code but couldn't quite understand how to customize buttons since now they are in a language i cannot understand. 
Thank you!

**Comment by AlexV525** [comment, marked as answer]
Sorry for didn't point it out in the README.

* If you only want to change the language/the text on buttons and other tips, use `AssetPickerTextDelegate` which is defined at https://github.com/fluttercandies/flutter_wechat_assets_picker/blob/master/lib/src/delegates/assets_picker_text_delegate.dart , then pass it through the pick method.
* If you are going to fully customize buttons, you can extends builder delegates here https://github.com/fluttercandies/flutter_wechat_assets_picker/blob/master/lib/src/delegates , then build your own pick method.

**Comment by dancb10** [comment]
Thank you for the response,
@AlexV525  can you pass a custom builder delegate to the AssetPicker.pickAssets method? From the docs I see you can pass AssetPickerTextDelegate but not a builder delegate. or how would this go if you want to build a custom builder delegate, how do you use it?
Thank you,
Dan

**Reply by AlexV525** [reply]
As I mentioned before, once you built your own delegate, you will need to build your own pick method. See https://github.com/fluttercandies/flutter_wechat_assets_picker/blob/master/example/lib/pages/custom_picker_page.dart#L35  for a implementation example.

**Reply by VladanRandjelovic** [reply]
Sorry to jump in with not necessarily related question, but i'm not quite getting how to change just the picker UI without creating additional providers?
To be more precise i want to make a UI that does not use navigator, but i want to use the default picker for images only,
would be more than thankful if you can explain how to do it, if it's possible anyhow.

Thanks in advance
